scripts/schedule_tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Create UUID for job creation / lookup
+uuid=$(uuidgen)
+
+# First, check that the type is set
+if [[ -z "${XLML_TEST_TYPE}" ]]; then
+    echo "Please set XLML_TEST_TYPE"
+    exit 1
+else
+    test_type="${XLML_TEST_TYPE}"
+fi
+
+# Loop over regions and connect to each cluster
+regions=(us-central1 us-central2 europe-west4)
+for region in ${regions[@]}; do
+    gcloud container clusters get-credentials xl-ml-test-"$region" --region "$region" --project xl-ml-test
+
+    # Loop over cron jobs, grepping for the test type
+    for i in $(kubectl get cronjobs -n automated -o name | grep "$test_type"); do
+        echo "$i"
```

Comment:
How do we prevent accidentally kick off many jobs matching to a short/trivial $test_type? Shall we add a confirmation step?

Reply:
Good call, I'll add a confirmation prompt before scheduling any jobs.